tests/python/pants_test/backend/jvm/tasks/jvm_compile/rsc/test_rsc_compile_integration.py

```diff
@@ -39,6 +39,22 @@ def test_basic_binary(self):
         'm.jar')
       self.assert_is_file(rsc_header_jar)
 
+  @ensure_compile_rsc_execution_strategy
+  def test_basic_binary_youtline(self):
+    with self.do_command_yielding_workdir('--compile-rsc-workflow=outline-and-zinc', 'compile', 'testprojects/src/scala/org/pantsbuild/testproject/mutual:bin') as pants_run:
+      zinc_compiled_classfile = os.path.join(
+        pants_run.workdir,
+        'compile/rsc/current/testprojects.src.scala.org.pantsbuild.testproject.mutual.mutual/current/zinc',
+        'classes/org/pantsbuild/testproject/mutual/A.class')
+      self.assert_is_file(zinc_compiled_classfile)
+      rsc_header_jar = os.path.join(
+        pants_run.workdir,
+        'compile/rsc/current/testprojects.src.scala.org.pantsbuild.testproject.mutual.mutual/current/rsc',
+        'm.jar')
+      import pdb
+      pdb.set_trace()
```

Comment:
You could combine 
`test_basic_binary` /  `test_basic_binary_youtline` and `test_rsc_hermetic_jvm_options ` /  `test_youtline_hermetic_jvm_options`
like
```
  @ensure_outliner
  @ensure_compile_rsc_execution_strategy
  def test_basic_binary(self):
    ...

  @ensure_outliner
  @ensure_compile_rsc_execution_strategy
  def test_hermetic_jvm_options(self):
    ...  
```